src/Lykke.Service.PayInvoice.Contract/Commands/RegisterEmployeeCommand.cs

```diff
@@ -0,0 +1,23 @@
+﻿using ProtoBuf;
+
+namespace Lykke.Service.PayInvoice.Contract.Commands
+{
+    [ProtoContract]
+    public class RegisterEmployeeCommand
+    {
+        [ProtoMember(1, IsRequired = true)]
+        public string MerchantId { get; set; }
```

Comment:
So for now the validation fields for RowKey, EmailAddress etc. is the responsibility of initializer of command?

Reply:
Good question, from my point of view the initializer can't be aware of all command executor's validation rules. I believe the command executor should have validation and generate the appropriate failed event in case validation is not passed.